.travis.yml

```diff
@@ -6,7 +6,7 @@ language: node_js
 node_js:
   - "6"
 script:
-  - bash ./scripts/deploy.sh
+  - bash ./src/scripts/deploy.sh
```

Comment:
We may want to keep the `/scripts` directory as is seeing as it's not part of the actual site code. @bebraw what do you think?